admin.tf

```diff
@@ -54,3 +64,77 @@ resource "aws_instance" "admin-ec2" {
         project             = "partinfra"
     }
 }
+
+resource "aws_security_group" "admin-elb-sg" {
+    name        = "admin-elb-sg"
+    description = "Admin ELB SG"
+    vpc_id      = "${aws_vpc.apps-shared-vpc.id}"
+}
+
+resource "aws_security_group_rule" "admin-elb-sg-allowhttp" {
+    type              = "ingress"
+    from_port         = 80
+    to_port           = 80
+    protocol          = "tcp"
+    cidr_blocks       = ["0.0.0.0/0"]
+
+    security_group_id = "${aws_security_group.admin-elb-sg.id}"
+}
+resource "aws_security_group_rule" "admin-elb-sg-allowhttps" {
+    type              = "ingress"
+    from_port         = 443
+    to_port           = 443
+    protocol          = "tcp"
+    cidr_blocks       = ["0.0.0.0/0"]
+
+    security_group_id = "${aws_security_group.admin-elb-sg.id}"
+}
+resource "aws_security_group_rule" "admin-elb-sg-allowall" {
+    type              = "egress"
+    from_port         = 0
+    to_port           = 0
+    protocol          = "-1"
+    cidr_blocks       = ["0.0.0.0/0"]
+
+    security_group_id = "${aws_security_group.admin-elb-sg.id}"
+}
+
+resource "aws_elb" "admin-elb" {
+  name                        = "admin-elb"
+  security_groups             = ["${aws_security_group.admin-elb-sg.id}"]
+  subnets                     = ["${aws_subnet.apps-shared-1c.id}"]
+  listener {
+    instance_port             = 80
+    instance_protocol         = "http"
+    lb_port                   = 80
```

Comment:
I suggest we redirect to ``https`` instead. I don't really like the idea of using jenkins with plain HTTP.